packages/flutter/test/cupertino/button_test.dart

```diff
@@ -7,56 +7,66 @@ import 'package:flutter/scheduler.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+const TextStyle testStyle = const TextStyle(
+  fontFamily: 'Ahem',
+  fontSize: 10.0,
+);
+
 void main() {
   testWidgets('Layout minimum size', (WidgetTester tester) async {
     await tester.pumpWidget(
-      new Center(child: new CupertinoButton(child: new Text(' '), onPressed: null))
+      new Center(child: new CupertinoButton(
+        child: new Text('X', style: testStyle),
+        onPressed: null,
+      ))
     );
     RenderBox buttonBox = tester.renderObject(find.byType(CupertinoButton));
     expect(
       buttonBox.size,
-      greaterThanOrEqualTo(const Size.square(48.0)),
-    );
-    expect(
-      buttonBox.size,
-      lessThan(const Size.square(100.0)),
+      // 1 10dp character + 16dp * 2 is smaller than the 48dp minimum.
```

Comment:
I don't really understand what that page is talking about (it has numbers without units, confusing acronyms, etc).

In flutter terminology, in the framework, all lengths are in logical pixels (px) and roughly speaking there are 96px to an inch unless a transform or other effect has been applied. In the engine, all lengths are in device pixels, or physical pixels, and exactly map to an actual LED or light or whatnot in the hardware.

The font is agnostic about all of this. I mean, technically it's all done in the engine and works in physical pixels, but you don't see that at the framework level where everything is logical pixels.